### ISSUE TITLE
Core: Placement (Rotation) using expressions has no "connection" with the object's property

### ISSUE THREAD

**Issue by kaiwas** [FreeCAD/FreeCAD#16983]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

If we set up rotation using the Placement tool with Euler angles, we do not see in the object's property that such a connection exists. At the same time, we can change the values ​​in the property (but they do not change to the values ​​we enter), while the values ​​in Placement remain unchanged.

Video demonstration. 

https://github.com/user-attachments/assets/4ccb6233-4a74-41a3-84d5-232d4df5a816

The test file is attached (remove the zip extension)
[Placement_EXP.FCStd.zip](https://github.com/user-attachments/files/17245509/Placement_EXP.FCStd.zip)

Perhaps it would be worth thinking about a switch for the rotation method in the object properties. This way we will have the usual (but inconvenient) quaternion method, but a new convenient one with Euler angles will appear.
Preliminary concept in the picture.

![Rottt](https://github.com/user-attachments/assets/146df7a3-c919-40ec-bf43-1bea63bb6fae)

### Full version info

```shell
OS: Manjaro Linux (KDE/plasma)
Word size of FreeCAD: 64-bit
Version: 1.1.0dev.38871 (Git) AppImage
Build type: Release
Branch: main
Hash: 813ee36cb80e0d29c2c9b991a7b2fee2c61f5cf6
Python 3.11.9, Qt 5.15.13, Coin 4.0.3, Vtk 9.2.6, OCC 7.7.2
Locale: Russian/Russia (ru_RU)
Stylesheet/Theme/QtStyle: OpenLight.qss/OpenLight/Qt default
Installed mods: 
  * lattice2 1.0.0
  * PitchedRoof
  * OpenTheme 2024.9.1
```

### Subproject(s) affected?

Expressions

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Roy-043** [comment]
Just FYI: the Euler angle expressions are (of course) listed  in the "Expression Engine" property (which is hidden by default).

**Comment by kaiwas** [comment]
> Just FYI: the Euler angle expressions are (of course) listed in the "Expression Engine" property (which is hidden by default).

Yes. Inconveniently, you can see that Euler angles are used. (See, but do not change)
But the quaternion input fields are still active and lead to an incomprehensible result. I think this is not quite correct.

**Comment by luzpaz** [comment]
@PaddleStroke is this something that is similar to the recent fix you made in Assembly (https://github.com/FreeCAD/FreeCAD/pull/17406) ?

**Comment by PaddleStroke** [comment]
Not really. I'm not sure how to do this since the object properties seems to be the quaternion.
@wwmayer this is a core issue that may interest you.

**Comment by kaiwas** [comment]
> Hi! This issue hasn’t seen activity in a while. If it’s still relevant, please update to the latest FreeCAD weekly build [download here](https://github.com/FreeCAD/FreeCAD-Bundle/releases/tag/weekly-builds) to see if the problem is resolved.

The problem is relevant

**Comment by kaiwas** [comment]
> Thanks for helping improve FreeCAD!

The problem remains.

**Comment by FEA-eng** [comment]
For some reason, when I open this file, "Euler" is renamed "0" and has empty Data properties:

<img width="1087" height="765" alt="Image" src="https://github.com/user-attachments/assets/e23b8252-6d67-45da-8346-1a3191bcb4c4" />

```
OS: Windows 10 build 19045
Architecture: x86_64
Version: 1.1.0dev.14555 (Git shallow)
Build date: 2025/09/30 23:55:55
Build type: Release
Branch: main
Hash: b08a0f87d68dd38cfcf39046209a7075adf11576
Python 3.11.13, Qt 6.8.3, Coin 4.0.3, Vtk 9.3.1, boost 1_86, Eigen3 3.4.0, PySide 6.8.3
shiboken 6.8.3, xerces-c 3.3.0, IfcOpenShell 0.8.2, OCC 7.8.1
Locale: Polish/Poland (pl_PL)
Navigation Style/Orbit Style/Rotation Mode: CAD/Rounded Arcball/Window center
Stylesheet/Theme/QtStyle: FreeCAD.qss/FreeCAD Light/
Logical DPI/Physical DPI/Pixel Ratio: 96/90.0633/1
```

**Comment by kaiwas** [comment]
> For some reason, when I open this file, "Euler" is renamed "0" and has empty Data properties:

I noticed that too.
The VarSet-name is renamed to "Zero."
But the object data seems to be there.

![Image](https://github.com/user-attachments/assets/7bb20d55-7c19-43c5-95a5-67f9235c3067)

**Comment by FEA-eng** [comment]
Now I see the Data properties, although the second VarSet is still named 0 for some reason:

<img width="391" height="456" alt="Image" src="https://github.com/user-attachments/assets/481fdc51-3bb9-47a9-bfab-5db07e48a61c" />

```
OS: Windows 10 build 19045
Architecture: x86_64
Version: 1.2.0dev.20260407 (Git shallow)
Build date: 2026/04/07 22:56:13
Build type: Release
Branch: (HEAD detached at 6db886fe0)
Hash: 6db886fe0e79bf060ffeaa50198da25702a17e31
Python 3.11.15, Qt 6.8.3, Coin 4.0.8, Vtk 9.3.1, boost 1_86, Eigen3 3.4.0, PySide 6.8.3
shiboken 6.8.3, xerces-c 3.3.0, IfcOpenShell 0.8.2, OCC 7.8.1
Locale: Polish/Poland (pl_PL)
Stylesheet/Theme/QtStyle: FreeCAD.qss/FreeCAD Light/
Navigation Style/Orbit Style/Rotation Mode: CAD/Rounded Arcball/Window center
Logical DPI/Physical DPI/Pixel Ratio: 96/90.0633/1
```
